build.gradle

```diff
@@ -782,6 +783,7 @@ project(':connect:runtime') {
     compile "org.eclipse.jetty:jetty-servlet:$jetty_version"
     compile "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:$jackson_version"
     compile "org.glassfish.jersey.containers:jersey-container-servlet:$jersey_version"
+    compile "org.reflections:reflections:$reflections_version"
```

Comment:
Having a worker running multiple connectors with conflicting versions of Guava will be so much fun!

I'll give implementing our own classpath search a go.